ZadanieDomowe7/Loops.cs

```diff
@@ -103,6 +116,7 @@ public static int Fibbonaci(int num)
 
             return Math.Sign(num) * fibbonaci;
```

Comment:
For Fibonacci you could not use negative numbers (at least based on your task). Please remove logic for negative numbers and throw exception in case of it